### PR TITLE
React decorator individualized

### LINKED ISSUE #2186: Remove Add Decorator from Global settings to Individual Component Settings - React
**User Story:**
As Spark we want to Remove Global Decorator from our Global Settings and place decorator into each Individual Component so that I can enhance individual components with out touching a global setting that touches all components.

**Notes:**
This is for React

**AC:** 
- Decorator will be removed from global setting
- Decorator will be added to all 22 individual components

### PR DESCRIPTION
**Pull request by RVMendoza**
## What does this PR do?
The container that surrounds each component is set globally, some components need individual changes, this allows for it.
- Add content to all masthead implementations -- you can now see scroll interactions because there's content to scroll.

### Associated Issue 
Fixes #2186 

### Code
 - [x] Build Component in Spark React

### Browser Testing (current version and 1 prior)
  - [x] Google Chrome
  - [x] Google Chrome (Mobile)
  - [x] Mozilla Firefox
  - [x] Mozilla Firefox (Mobile)
  - [x] Microsoft Internet Explorer 11 (only this specific version of IE)
  - [x] Microsoft Edge
  - [x] Apple Safari
  - [x] Apple Safari (Mobile)

### CODE REVIEW

File: angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead.stories.ts [hunk 1]
```diff
@@ -231,6 +231,107 @@ export const defaultStory = () => ({
         </a>
       </div>
     </sprk-masthead>
+    <div class="sprk-u-pam">
+      <p class="sprk-u-mbm">
+        Molestie convallis gravida neque fermentum
```
Comment (afebbraro) [inline angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead.stories.ts]: Hey @RVMendoza, these paragraphs are not part of the masthead component and having them there is a bit confusing to users. They show up in the copy code block too, so they are copying a wall of text that isn't related to the component.

File: react/src/components/footer/SprkFooter.stories.js [hunk 2]
```diff
@@ -201,14 +211,14 @@ const awards = {
     {
       href: '#nogo',
       src:
-        'https://spark-assets.netlify.com/spark-logo.svg',
+        'https://spark-assets.netlify.com/spark-logo-black-text.svg',
```
Comment (afebbraro) [inline react/src/components/footer/SprkFooter.stories.js]: Whats the image change?

File: react/src/components/masthead/SprkMasthead.stories.js [hunk 3]
```diff
@@ -98,247 +116,440 @@ const utilityItems = [
...
-    utilityContents={utilityItems}
-    navLink={<SprkLink variant="simple" href="#nogo" additionalClasses="sprk-c-Masthead__link">Sign In</SprkLink>}
-  />
+    </svg>}
+      utilityContents={utilityItems}
+      navLink={<SprkLink variant="simple" href="#nogo" additionalClasses="sprk-c-Masthead__link">Sign In</SprkLink>}
+    />
+    <div class="sprk-u-pam">
+      <p class="sprk-u-mbm">
+        Molestie convallis gravida neque fermentum
```
Comment (afebbraro) [inline react/src/components/masthead/SprkMasthead.stories.js]: See the comment about this extra text up above on the angular one.

File: react/src/components/masthead/SprkMasthead.stories.js [hunk 4]
```diff
@@ -98,247 +116,440 @@ const utilityItems = [
...
-    utilityContents={utilityItems}
-    variant="extended"
-  />
+    </svg>}
+      utilityContents={utilityItems}
+      variant="extended"
+    />
+    <div class="sprk-u-pam">
+      <p class="sprk-u-mbm">
+        Molestie convallis gravida neque fermentum
```
Comment (afebbraro) [inline react/src/components/masthead/SprkMasthead.stories.js]: See comment up above on angular masthead about this text.

File: angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead.stories.ts (hunk 1 above)
Comment (RVMendoza) [reply angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead.stories.ts]: There may be some better way, but I feel having content here best represents the masthead component and all of it's features.

One of the things I was thinking about when making masthead is that I couldn't test whether or not scrolling features worked on small viewport. I added this content to be able to test that.

Then I removed it.

Later, I had to test it one more time and had to add the content again.

I understand the sb is for devs to test and work on, as well as an implementer.

If I take this out, it feels weird to me that a user cannot test to see what masthead does on small viewport scrolling. It also feels weird that as a developer, anytime I need full access to everything masthead does, I HAVE to add fake content before I develop.

Maybe we can add this as a knob later on? Body content knob?

File: react/src/components/footer/SprkFooter.stories.js (hunk 2 above)
Comment (RVMendoza) [reply react/src/components/footer/SprkFooter.stories.js]: it makes it black and readable instead of white text

**Comment by RVMendoza** [comment]
@afebbraro changes were done!

File: react/src/components/masthead/SprkMasthead.stories.js [hunk 5]
```diff
@@ -4,6 +4,24 @@ import SprkButton from '../buttons/SprkButton';
 import SprkLink from '../../base/links/SprkLink';
 import SprkDropdown from '../dropdown/SprkDropdown';
 
+export default {
+  title: 'Components|Masthead',
+  component: SprkMasthead,
+  decorators: [
+    story => <div>{story()}</div>
```
Comment (afebbraro) [inline react/src/components/masthead/SprkMasthead.stories.js]: What does the masthead need this <div> decorator for?

File: angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead.stories.ts (hunk 1 above)
Comment (afebbraro) [reply angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead.stories.ts]: thanks for removing these @RVMendoza !

File: react/src/components/masthead/SprkMasthead.stories.js (hunk 5 above)
Comment (afebbraro) [reply react/src/components/masthead/SprkMasthead.stories.js]: left this question on Friday @RVMendoza

Comment (RVMendoza) [reply react/src/components/masthead/SprkMasthead.stories.js]: I figured they all needed them, which sets them to be available to change if it ever comes up.

I can remove it.

**Comment by afebbraro** [comment]
@RVMendoza i approved this but it has conflicts that need to be resolved before I can merge it.

**Pull request merged**